src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -134,16 +149,43 @@ public void visit(NodeTraversal t, Node n, Node parent) {
           // We export the longest valid prefix
           String exportedNamespace = nameUtil.findLongestNamePrefix(lhs, symbols.keySet());
           if (exportedNamespace != null) {
-            convertExportAssignment(child, exportedNamespace, symbols.get(exportedNamespace));
+            convertExportAssignment(
+                child, exportedNamespace, symbols.get(exportedNamespace), filename);
             // Registers symbol for rewriting local uses
-            registerLocalSymbol(child.getSourceFileName(), exportedNamespace, exportedNamespace,
+            registerLocalSymbol(
+                child.getSourceFileName(),
+                exportedNamespace,
+                exportedNamespace,
                 symbols.get(exportedNamespace));
           }
           break;
         default:
           break;
       }
     }
+
+    private void directlyExportSymbol(Node namedNode, Node parent, String filename) {
```

Comment:
please camel case as `fileName`, to be consistent with JSCompiler's API (here and everywhere).